lib/ProductOpener/Store.pm

```diff
@@ -368,12 +387,35 @@ sub retrieve_object_json($path) {
 	return $json_for_objects->encode(retrieve($path . '.sto'));
 }
 
+
+=head2 object_exists($path)
+
+Indicates whether an object (STO or JSON) exists at the specified path
```

Comment:
As far as I can tell: `object_exists` is used solely for `change_product_code`.  I am trying to think of any edge cases around this (e.g. for some reason a `.sto` file exists where we have not yet written a JSON file -- so we would consider that the object does exist, but we have not migrated it to the latest file format yet).